extra/release_tool.py

```diff
@@ -2407,6 +2414,39 @@ def do_build(args):
     trigger_build(state, tag_avail)
 
 
+def determine_version_bump(state, repo, from_v, to_v):
+    cmd = ["conventional", "list-commits", "--from", from_v, "--to", to_v, "--parse"]
```

Comment:
Q: When this is already a `Python` tool, why do we need to use the CLI here? Can we not use this as a library inline?

Reply:
It seems they're using a lib internally: https://github.com/multimac/conventional/blob/master/conventional/parser/conventional_commits.py

Reply:
Not really... The cli tool does a bunch of stuff that is really useful like fetching all relevant commit messages, preprocessing them for the parser, reading a default config file etc. I agree that using a library directly would be optimal, but I believe its only possible with a hacky approach, since it is not really meant to be used that way.

The linked module in your comment is not really meant to be imported. The cli will read a [yml config file](https://github.com/multimac/conventional/blob/master/conventional/config_default.yaml) where one of the fields in the parser field is a module referencing to `ConventionalCommitParser` meaning that a user is able to replace the default parser if needed.